src/data/alumni.ts

```diff
@@ -0,0 +1,152 @@
+  },
+  {
+    name: 'Aaditya Mehta',
+    graduation: 2021,
+    description:
+      'Aaditya studied Economics and Mathematics and spent way too much time in Stern (#IAmStern). In his free time he likes to listen to music, take long walks, and read Econ papers, sometimes simultaneously. Aaditya was the former treasurer.',
+  },
+  {
+    name: 'Sonia Jain',
+    graduation: 2022,
+    description:
+      "Sonia double majored in Computer Science and Media, Culture, and Communications. She is interested in game design, art, and I love animals. In her free time, Sonia works with a variety of organizations to help find dogs and cats' homes. She also loves painting and digital art so when she is not working with animals, she is creating. Sonia was the former director of marketing & communications.",
+  },
+  {
+    name: 'Bruce Jiang',
+    graduation: 0,
+    description:
+      'Bruce is from China and studied Computer Science and Mathematics. Outside of class, he is crazy in soccer (especially in Chelsea) and basketball. He spends tons of time playing FIFA (most of the time playing Chelsea). Besides, he likes listening to music, reading about history, and travelling. Bruce was the former secretary.',
+  },
+  {
+    name: 'Evan Silverman',
+    graduation: 2022,
+    description:
+      'Evan studied CS at CAS. In his free time, he enjoys working on too many side projects, expanding his record collection, and making memes for @nyumsfbt. Evan was a former full stack developer.',
+  },
+  {
+    name: 'Yuchen Liu',
+    graduation: 2023,
+    description:
+      'Yuchen studied CS and Data Science at CAS. He likes to eat ramen, watch movies, meet new people, and work on fun projects. Yuchen was a former full stack developer.',
+  },
+  {
+    name: 'Samuel Lin',
+    graduation: 2021,
+    description:
+      'Samuel was at CAS majoring in Computer Science and minoring in Web Design and Game Design. He was born and raised in Taiwan, but studied in Singapore for high school before coming to New York. He is a big foodie and enjoys trying foods from different cultures. Samuel was a former event coordinator.',
+  },
+  {
+    name: 'Vishal Vanam',
+    graduation: 2022,
+    description:
+      'Visuhal was a Graduate Student at Information Systems at GSAS. He hails from Hyderabad (India) and love traveling and exploring new places. He is usually spotted listening to music and engaging in debates with friends. He was a former event coordinator.',
+  },
+  {
+    name: 'Nadira Dewji',
+    graduation: 2018,
+    description:
+      'Nadira was one of the Founders of BUGS and studied Computer Science in CAS. Originally from Tanzania, Nadira loves travelling, scuba diving and wildlife. She’s excited about open source principles and making impactful contributions.',
```

Comment:
```suggestion
      "Nadira was one of the Founders of BUGS and studied Computer Science in CAS. Originally from Tanzania, Nadira loves travelling, scuba diving and wildlife. She's excited about open source principles and making impactful contributions.",
```
Not sure if this is from the formatter but I think using a regular apostrophe is better than the unicode for quotation mark (U+2019). Or if we want to stick with the unicode we should change all the other apostrophes to the unicode mark.